### PR TITLE
Bump Rosetta spec

### PR DESCRIPTION
**Pull request by bisgardo**
## Purpose

Bump Rosetta spec to the current version.

## Changes

Bumping from [v1.4.12 to v1.4.15](https://github.com/coinbase/rosetta-specifications/compare/v1.4.12...v1.4.15). Don't mind that the spec refers to itself as v1.4.13; it's build from the commit tagged v1.4.15 ([proposed fix](https://github.com/coinbase/rosetta-specifications/pull/115)). Also don't mind the breaking changes that happened in these patch version bumps...

The bump was performed by checking out tag `v1.4.15` in `./rosetta-specifications` and then running
```shell
npm install
npm run generate
```
from the project root.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,14 +1,1222 @@
 {
```
Comment (abizjak) [inline package-lock.json:1]: Why is this file now committed? I cannot see any reference to it, or its explanation. That needs to be included if this is to remain.

**Review by abizjak — COMMENTED** [review]
What prompted this update which seems to have some breaking features ( as pointed out in the PR description)?

File: package-lock.json (hunk 1 above)
Comment (bisgardo) [reply package-lock.json:1]: It's not a new file. I'm not sure why it got so much larger but as `lockfileVersion` was bumped that's probably the reason.

**Comment by bisgardo** [comment]
> What prompted this update which seems to have some breaking features ( as pointed out in the PR description)?

Just a desire to stay on the most recent version of the spec. Found out randomly that we were a few versions behind and that it was a trivial upgrade so decided to go for it.

The breaking changes don't break anything in our impl. The only change is that you no longer get errors for setting a few fields that used to be in the spec (but were unused by us) that are no longer in the spec.

File: package-lock.json (hunk 1 above)
Comment (bisgardo) [reply package-lock.json:1]: The file is managed by npm so the change was automatic.

**Review by rasmus-kirk — APPROVED** [review]

**Pull request merged**